lib/unleash_proxy_client_flutter.dart

```diff
@@ -147,6 +156,8 @@ class UnleashClient extends EventEmitter {
       this.headerName = 'Authorization',
       this.customHeaders = const {},
       this.impressionDataAll = false,
+      // should be used only for testing
+      this.userAgent = 'unleash-flutter@1.9.0',
```

Comment:
So this is the one we need to manually update? Why does it say it should only be used for testing? Maybe we should be even clearer with the comments and add a note to say you should update this when releasing?

Reply:
3 things. we update it on each release. we overwrite it in tests for predictability. users should never user it themselves

Reply:
making the comment more explicit